server_environment.yml

```diff
@@ -3,7 +3,7 @@ channels:
 - conda-forge/label/vtk_dev
 - conda-forge
 dependencies:
-- python>=3.7
+- python ==3.8
```

Comment:
why the pinning?

Reply:
segfaults on 3.9 for unknown reasons

Reply:
Ouch. Has this been reported upstream?

Reply:
No, I haven't tried to replicate locally or anything. Feel free if you have time

Reply:
Also, do we actually need the VTK dev channel any longer?

Reply:
Not sure, maybe that's the problem with the bump to 3.9

Reply:
Will have a look.